### ISSUE TITLE
Kubernetes: Network Policies

### ISSUE THREAD

**Issue by steven-sheehy** [hashgraph/hedera-mirror-node#655]
**Problem**
Kubernetes [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies) isolate pods network traffic and are considered best practice to implement.

**Solution**
- Enable network policy in the cluster (requires a special CNI)
- Enable network policy included in dependent charts
- Write network policies for the apps we control
- Write network policies for apps we don't control and submit to upstream

**Alternatives**

**Additional Context**

**Comment by rbair23** [comment]
I found [some documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy) about setting up a GKE k8s cluster to use network policy. It [looks like](https://kubernetes.io/docs/tasks/administer-cluster/network-policy-provider/calico-network-policy/) it uses Calico by default.

**Comment by steven-sheehy** [comment]
Yeah, it's enabled in our GKE cluster we just haven't written the network policies. Though GKE Autopilot, which we're hopefully transitioning to, currently does not support Network Policies.
